gef-ui/templates/partials/application-preview.njk

```diff
@@ -31,8 +31,13 @@
   {% if not isAutomaticCover %}
     {% include "includes/application-details/supporting-information.njk" %}
   {% endif %}
-  {% if checkerCanSubmit %}
-    {% include "includes/application-preview/submit.njk" %}
+
+  {% if coverDatesConfirmed and (applicationStatus == 'UKEF_APPROVED_WITHOUT_CONDITIONS' or applicationStatus == 'UKEF_APPROVED_WITH_CONDITIONS') %}
```

Comment:
can we do this in the server and then in the template can just be 

`{% if makerCanSubmit % }` or checker whoever the user is

Maybe worth adding a check that the user has correct role

Reply:
Done